static/templates/settings/notification_settings.hbs

```diff
@@ -99,6 +100,7 @@
 
         <div class="subsection-header inline-block">
             <h3>{{t "Mobile message notifications" }}</h3>
+            <a href="/help/mobile-notifications" target="_blank"><i class="fa fa-question-circle-o settings-info-icon"></i></a>
```

Comment:
You can use `help_link_widget` template instead of this. `git grep help_link_widget static/templates` would be helpful to know how it is use for other links.

Reply:
Thankyou Sahil, I have updated the changes according to the `help_link_widget`.